api/v1/azureappconfigurationprovider_types.go

```diff
@@ -141,8 +141,8 @@ type ManagedIdentityReferenceParameters struct {
 	Key string `json:"key"`
 }
 
-// AzureKeyVaultReference defines the authentication type used to Azure KeyVault resolve KeyVaultReference
-type AzureKeyVaultReference struct {
+// SecretReference defines the authentication type used to Azure KeyVault resolve KeyVaultReference
```

Comment:
The description can be updated to match the latest usage. It contains more supported properties than auth.

Reply:
Updated